uptycs/resource_event_rule.go

```diff
@@ -119,9 +119,10 @@ func (r *eventRuleResource) Schema(_ context.Context, req resource.SchemaRequest
 				Optional: true,
 				Attributes: map[string]schema.Attribute{
 					"enabled": schema.BoolAttribute{
-						Computed: true,
```

Comment:
`Computed` means its "read only" and you cant change it, hence the error. This is more for things like `id` where its returned and you want to capture it to reference (`uptycs_thing.foo.id` etc)